build-release.sh

```diff
@@ -8,7 +8,7 @@ eval "$(go env)"
 VERSION=${1:-$(git describe --tags --always --dirty)}
 
 # To overwrite the version details, pass something as the second arg. Empty string disables it.
-VERSION_DETAILS=${2-"$(date --utc --iso-8601=s)/$(git describe --always --long --dirty)"}
+VERSION_DETAILS=${2-"$(date -u +"%Y-%m-%dT%H:%M:%S%z")/$(git describe --always --long --dirty)"}
```

Comment:
can you make it `%:z` in order for it to be exactly as the current one

Reply:
@MStoykov we can't, BSD date does not support `%:z`